.brigade/project.yaml

```diff
@@ -0,0 +1,18 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/brigadecore/brigade/v2/v2/apiserver/schemas/project.json
+apiVersion: brigade.sh/v2-alpha.5
+kind: Project
+metadata:
+  id: brigade
+description: Brigade 2 built with Brigade 2!
+spec:
+  eventSubscriptions:
+  - source: brigade.sh/github
+    qualifiers:
+      repo: brigadecore/brigade
+    types:
+    - check_run:rerequested
+    - check_suite:requested
+    - check_suite:rerequested
```

Comment:
(see request to handle `issue_comment` events in the `brigade.ts` file; if we do so, we'd add the corresponding types here)

Reply:
[Same answer](https://github.com/brigadecore/brigade/pull/1461#discussion_r660995142). Gateway takes care of it.